cmd/podman/generate_kube.go

```diff
@@ -36,20 +36,21 @@ var (
 // generateKubeYAMLCmdgenerates or replays kube
 func generateKubeYAMLCmd(c *cli.Context) error {
 	var (
-		container *libpod.Container
-		err       error
-		output    []byte
+		podYAML        *v1.Pod
+		container      *libpod.Container
+		err            error
+		output         []byte
+		pod            *libpod.Pod
+		mashalledBytes []byte
+		servicePorts   []v1.ServicePort
 	)
 
 	if rootless.IsRootless() {
 		return errors.Wrapf(libpod.ErrNotImplemented, "rootless users")
```

Comment:
not part of this PR, but I think it is safe to drop this check.  Rootless containers should just work.